### ISSUE TITLE
Piracy Interview

### ISSUE THREAD

**Issue by ZachAndrews98** [nadeld/nadeld.github.io#5]
I would go back and work on the wording of the first half of your introduction, it doesn't really make much sense and is a little hard to follow.
Not sure you should give the name of the interviewee since they have "plenty of experience pirating games".
You are restating your questions and giving the responses your interviewee gave, the assignment sheet says not to do this.

**Comment by nadeld** [comment]
Thanks Zach!
